apps/web/lang/en.json

```diff
@@ -171,7 +171,7 @@
   "quantitySelectorIncrease": "Increase value",
   "scrollTop": "Scroll to top",
   "numberInCart": "{count} product in cart",
-  "languageSelector": "Language selection",
+  "languageSelector": "Click to open languageSelector modal",
```

Comment:
Why are we using a new translation here?

Reply:
if you search "languageSelector" in project this word is not use, so i changed it to match provided mockups

Reply:
![Screenshot 2023-11-17 at 11 53 24](https://github.com/plentymarkets/plentyshop-pwa/assets/121931658/0b41e60d-089f-4d05-98df-6c171667409e)